### DISCUSSION TITLE
Is that possible to delete policies.json in filer server?

### DISCUSSION THREAD

**Discussion by guo-sj** [seaweedfs/seaweedfs#2617] [Q&A] [answered]
I uploaded a `policies.json` file into filer server path "/etc/iam" by mistake, **is any possible to delete that file**?

I had tried to use `curl -X DELETE http://localhost:26000/etc/iam` (my filer server is running in port 26000), and got this response:
```
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>MethodNotAllowed</Code><Message>The specified method is not allowed against this resource.</Message><Resource>/etc/iam/policies.json</Resource><RequestId>1643186570074047257</RequestId></Error>#
```

And If I don't delete `policies.json` which I uploaded, the `iam server` will exit when I send a `create policy` request by aws CLI.
The request command is:
```
$ aws --endpoint-url http://localhost:8111 iam create-policy --policy-name my-policy --policy-document file://policy
```

Any help will be appreciated.

**Comment by aoberest** [comment, marked as answer]
Hello,
Do you try to use weed shell ? 

`echo "fs.cat /etc/iam/policies.json" | weed shell` 
Does this command show the contents of a file?

**Reply by chrislusf** [reply]
The `weed shell` also supports deletion: `fs.rm`.

**Reply by chrislusf** [reply]
One more undocumented trick: you can skip the "fs." part, for all "fs.*" commands.

```
> ls
dd.dat
topics
> ls -al topics
drwxr-xr-x   0 chrislu staff      0 /topics/.system
total 1
```

**Reply by guo-sj** [reply]
Thanks, the problem is gone by using `weed shell`.
